ARGO/src/FiniteStateMachine.cpp

```diff
@@ -0,0 +1,52 @@
+#include "stdafx.h"
+#include "FiniteStateMachine.h"
+#include "Idle.h"
+
+FiniteStateMachine::FiniteStateMachine()
+{
+	m_current = new Idle();
+}
+
+FiniteStateMachine::~FiniteStateMachine()
+{
+}
+
+void FiniteStateMachine::setCurrent(State* s)
+{
+	m_current = s;
+}
+
+void FiniteStateMachine::setPrevious(State* s)
+{
```

Comment:
s parameter name should be changed to 'state' or equivalent and t_

Reply:
fixed